pyigm/guis/igmguesses.py

```diff
@@ -827,7 +865,8 @@ def add_component(self, inp, vlim=None, zcomp=None, update_model=True):
                 linelist_aux = self.llist[current_parent_linelist]
 
             # add the component
-            new_comp = create_component(zcomp, inp, linelist_aux, vlim=vlim, spec=self.spec)
+            new_comp = create_component(zcomp, inp, linelist_aux, vlim=vlim, spec=self.spec,
+                                        spwvmin=self.parent.wvmin, spwvmax=self.parent.wvmax)
```

Comment:
Same as above.  Why are we still referencing the parent `wvmin` ?

Reply:
Checked again. For the same or similar reason as above. When igmguesses is used with -p guesses.json -o guesses.json option, calling self.spec.wvmin and self.spec.wvmax does the following: When a new absorption component is defined, the absorption component contains only absorption lines within the wavelength range of a spectrum. (i.e., within [self.spec.wvmin, self.spec.wvmax]). On the other hand, calling self.parent.wvmin and self.parent.wvmax includes all absorption lines in a customly defined wavelength range [wmin,wmax]. In this way, one guesses file could be used when working with multiple non-overlapping spectra (or spectra with different wavelength ranges) of the same object. Therefore, it should be self.parent.wvmin and self.parent.wvmax.